aws/dex-roles/main.tf

```diff
@@ -268,3 +268,63 @@ resource "aws_iam_role_policy_attachment" "dex_automation_role_attachment" {
   role       = aws_iam_role.dex_automation.name
   policy_arn = aws_iam_policy.dex_automation_policy.arn
 }
+
+resource "aws_athena_workgroup" "dbt_sql_prod" {
+  name = "dbt_sql_prod"
```

Comment:
We usually name AWS resources using `-` rather than `_`. Could we change this or is this a requirement?

Reply:
fixed!